scripts/rqt_pr2_dashboard

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
```

Comment:
Same as in https://github.com/PR2/linux_networking/pull/5#discussion_r584112906 .
Please make sure the file is correctly installed via `catkin_install_python` instead and test whether it still works.

Reply:
fixed